cfme/tests/openstack/cloud/test_tenant_crud.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from fauxfactory import gen_alphanumeric
+import cfme.fixtures.pytest_selenium as sel
+from cfme.cloud.tenant import Tenant
+from utils.wait import wait_for
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_create_cloud_tenant(provider):
+    """
+        create a new cloud tenant
+    """
```

Comment:
Can you follow some reasonable test descriptions?
Like this one https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/test_ui_problems.py#L26